.github/workflows/install.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - master
+      - "pr-**"
```

Comment:
Please remove all these changes. I really appreciate your contributions, but I'm really tired of telling you to remove unrelated stuff from your PRs :(